src/libs/actions/IOU.ts

```diff
@@ -1578,6 +1590,12 @@ function buildOnyxDataForTrackExpense(
         },
     );
 
+    if (iouReport) {
```

Comment:
iou reports (p2p reports) shouldn't have report violations

These should be exclusive to expense reports that are on control policies. Though I can't tell if iouReport in this context means something else

Reply:
There is a comment below specifying when we should compute violations: https://github.com/Expensify/App/blob/f7e265a65f695edd85870bd0c8048cf4578e14fb/src/libs/actions/IOU.ts#L874

Should I move my code under this condition?

cc @thienlnam

Reply:
Yeah that would be good, and if you could make sure that the paid policy is a control policy

Reply:
`isPaidGroupPolicy` checks if a policy is of type `CORPORATE` or `TEAM`

![image](https://github.com/user-attachments/assets/74d9f82d-fcbd-4ac7-ae23-17762f6f56b0)

and those are control and collect workspaces which are paid

![image](https://github.com/user-attachments/assets/462c499f-6ccb-40a9-96f9-3e83b2a9c40f)

Reply:
Can you check specifically for corporate policies? We should not show these for team either

Reply:
Done!